tcms/testruns/views.py

```diff
@@ -259,26 +260,21 @@ def delete(request, run_id):
 
 
 @require_GET
-def all(request, template_name='run/all.html'):
+def get_all(request):
     """Read the test runs from database and display them."""
     query_result = len(request.GET) > 0
 
     if request.GET:
         search_form = SearchRunForm(request.GET)
-        if request.GET.get('product'):
-            search_form.populate(product_id=request.GET['product'])
-        else:
-            search_form.populate()
-
-        search_form.is_valid()
```

Comment:
`is_valid` does not seem to do anything. It just returns a result, showing if the form is valid or not. Since we do not use this result anywhere I suppose the call to `is_valid` is obsolete, since it does not make any change to the state of `search_form`

Reply:
This call performs form validation but the caller doesn't check the result. See other places in the code. Bring back this line and open an issue for dealing the not checking the result properly.